cmd/dynatrace/dtquery.go

```diff
@@ -18,6 +19,17 @@ func (q *DTQuery) InitLogs(hours int) *DTQuery {
 	return q
 }
 
+func (q *DTQuery) InitLogsWithTimeRange(from time.Time, to time.Time) *DTQuery {
+	q.fragments = []string{}
+
+	fromStr := from.Format("2006-01-02T15:04:05Z")
```

Comment:
Would it make sense to define the format string only once as a constant and reference that?